NEWS.md

```diff
@@ -58,7 +58,7 @@ for human readability.
   argument now comes first.
 - The undocumented and unused
   `DGMultiMesh(triangulateIO, rd::RefElemData{2, Tri}, boundary_dict::Dict{Symbol, Int})`
-  constructor was removed.
+  constructor was deprecated.
```

Comment:
We should change the base of this PR to the v0.6 development branch (I will do so in a minute). Then, please undo this change here and lit it as a change in a newer section describing how to upgrade to v0.6 from v0.5.x.